md/ui-designer-overview.md

```diff
@@ -57,3 +57,24 @@ To use a layout in a business application, export it from the Bonita UI designer
 
 The UI Designer is intended for web developers creating pages and forms that will be part of a BPM application. It requires a good understanding of JavaScript and CSS.  
 If you are a business analyst, you can use the UI Designer to create the page structure and define basic behaviors. However, most properties of a page are defined using JavaScript.
+
+## UI Designer with collaboration
+
+Id of Pages or forms who are created in 7.6 older version is an uuid.
```

Comment:
Pages which have been created with a version of bonita older than 7.7.0 have an UUID-like identifer.